pkg/eestream/aesgcm.go

```diff
@@ -128,3 +130,43 @@ func (s *aesgcmDecrypter) Transform(out, in []byte, blockNum int64) (
 
 	return s.aesgcm.Open(out, n[:], in, nil)
 }
+
+// EncryptAESGCM encrypts byte data with a key and nonce. The cipher data is returned
+func EncryptAESGCM(data, key, nonce []byte) (cipherData []byte, err error) {
+	block, err := aes.NewCipher(key)
+	if err != nil {
+		return []byte{}, errs.Wrap(err)
+	}
+	aesgcm, err := cipher.NewGCM(block)
+	if err != nil {
+		return []byte{}, errs.Wrap(err)
+	}
+	cipherData = aesgcm.Seal(nil, nonce, data, nil)
+	return cipherData, nil
+}
+
+// DecryptAESGCM decrypts byte data with a key and nonce. The plain data is returned
+func DecryptAESGCM(cipherData, key, nonce []byte) (data []byte, err error) {
+	if len(cipherData) == 0 {
+		return []byte{}, errs.New("empty cipher data")
+	}
+	if err != nil {
+		return []byte{}, errs.Wrap(err)
+	}
+	if err != nil {
+		return []byte{}, errs.Wrap(err)
+	}
```

Comment:
What are these two err checks testing above? There wasn't any function call returning an error yet.

Reply:
It was probably a mistake I didn't catch. I'll remove them